packages/paste-core/components/modal/src/utils/consoleUtils.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * HACK NOTICE:
+ * This is an eggregious hack but currently the best way for us
+ * to maintain accessibility in our modal and keep it functional
+ * in Twilio's Console application.
+ *
+ * For more info, see: https://github.com/reach/reach-ui/issues/610
+ *
+ * FIXME: when the Console app redesigns navigation
+ */
+
+// 80px because the issue is the `calc(100vh - 80px)`
+const MARGIN_AMOUNT_BASED_ON_CALC_OFFSET = '80px';
+
+export function addConsoleHeightPatch(): void {
+  if (document == null) {
+    return;
+  }
+
+  const CONTENT_WRAPPER = document.querySelector('#content') as HTMLElement;
+  const SIDEBAR_WRAPPER = document.querySelector('#sidebar-wrapper') as HTMLElement;
```

Comment:
Given that the reason this happens in Console is reach modal removed padding top from the `body`, was there a particular reason you aren't just adding the 80px padding-top back to the body, rather than adding margin to 2 other elements?

Reply:
As discussed on Slack, this is for more specificity for future maintainers.